en/docs/references/define-configurable-variables.md

```diff
@@ -0,0 +1,126 @@
+   
+        !!! Info
+            To learn how to create a REST API with more details, try the [Create Your First REST API tutorial](../tutorials/create-your-first-rest-api.md).
+
+    4. Once the **Components** page opens, click **Edit Code** to open the Web Editor.
+
+2. Now let's define the email address you want to return as a configurable by following these sub-steps:
+
+    1. Click the first **+** icon on the page, and in the **Add Constructs** pane that appears, click **Configurable**.
+   
+        ![Add Configurable](../assets/img/configurables/add-configurable.png){.cInlineImage-threeQuarters}
+   
+        The **Configurable** pane opens, displaying the template for the configurable.
+    
+    2. The **Configurable** pane displays the configurable template. Let's use this template and define the configurable.
+   
+        1. By default, the data type of the configurable variable is `int`. To change it to a different type, click **`int`**, and click the required type in the list of suggestions. For this guide, click **string**.
+       
+        2. To enter a name for the configurable variable, double-click **`<add-config-name>`** and enter the required text (e.g., `email`).
+       
+        3. By default, the value of the configurable variable is **`?`**. This value specifies that the configurable field should appear as an empty field at deployment. For this guide, let's leave it unchanged.
+
+        4. Click **Save**. The Web Editor displays the added configurable as shown in the image below:
+
+            ![Added Configurable](../assets/img/configurables/added-configurable.png){.cInlineImage-half}
+   
+3. Now let's update the last return statement to return the configurable variable you added instead of the default greeting. To do this, follow these sub-steps:
+
+    1. Click the edit icon on the last return statement (shown in the image below).
+
+        ![Edit return statement](../assets/img/configurables/edit-return-statement.png){.cInlineImage-half}
+
+        This **Return** pane opens, displaying the existing return statement.
+
+    2. Click **`name`** and then click **Expressions**.
+   
+    3. Under **Arithmetic**, click **Es + Ex**.
+   
+    4. Double-click **`add-expression`**, and enter `"Please email your application to "`.
+   
+    5. Click **Es + Ex** again, and then click **Suggestions**.
+   
+    6. In the list of suggestions, click **email**. Here, you are selecting the configurable variable that you previously added.
+
+    7. Click **Save**.
+   
+        You have completed editing the REST API configuration.
+
+4. To view the Pro-Code view of the REST API, click the icon marked in the image below:
+
+    ![Open pro-code view](../assets/img/configurables/open-code-view.png){.cInlineImage-full}
+
+    You will see the configurable you added in the code view as `configurable string email = ?;`.
```

Comment:
Should we go with the term code view in step 4 as well? We seem to be using the term "Pro-Code view" there.

Reply:
Changed to "code view".